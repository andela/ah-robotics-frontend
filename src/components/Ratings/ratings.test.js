import React from 'react';
import RatingComponent from './index';

const prop = {
    ratings: {
        userRating: {},
        error: {},
        averageRating: 0,
        isLoading: false,
        isSuccess: false,
        isUpdated: false,
        isError: false,
        rating: {
            message: {},
            data: {
                article: 'this-is-africa',
                average_rating: 2.5,
                user_rating: 4,
            },
        },
    },
};
const square = 0;
const handleRate = () => {};

it('app should match the snapshot', () => {
    const wrapper = shallow(
      <RatingComponent prop={prop} square={square} handleRate={handleRate} />,
);
    expect(wrapper).toMatchSnapshot();
    });
