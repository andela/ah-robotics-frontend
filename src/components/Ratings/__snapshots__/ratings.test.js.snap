// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match the snapshot 1`] = `
<div
  className="ratings-container"
>
  <Grid
    style={
      Object {
        "height": "100%",
      }
    }
    textAlign="center"
    verticalAlign="middle"
  >
    <GridColumn
      className="form-container"
    >
      <Segment
        circular={true}
        style={0}
      >
        <Header
          as="h2"
        >
          Rating!
          <HeaderSubheader />
        </Header>
        <Rating
          clearable="auto"
          icon="star"
          maxRating={5}
          onRate={[Function]}
        />
      </Segment>
    </GridColumn>
  </Grid>
</div>
`;
